wgsl/index.bs

```diff
@@ -6811,11 +6811,14 @@ That's not a full user-defined function declaration.
     (GLSLstd450Fract)
 
   <tr algorithm="frexp">
-    <td>|T| is [FLOATING]<br>
+    <td>|T| is [FLOATING];<br>
         |I| is [SIGNEDINTEGRAL], where<br>
         |I| is a scalar if |T| is a scalar, or<br>
-        a vector when |T| is a vector<br>
-        |SC| is a storage class<br>
+        a vector when |T| is a vector;<br>
+        |SC| is storage class 
```

Comment:
why aren't we allowed to pass pointers to buffer contents here?

Reply:
The weasely answer is that WGSL has the restriction that pointer arguments can only be from these three storage classes, and this repeats that constraint. (Because a builtin function is a function.)

But with #1850 we need to relax that constraint to allow for arrayLength and the atomic builtin functions.  And so we return to your question.

The slightly less weasely answer is that @ben-clayton requested it.  Maybe he can motivate it.